libs/blocks/quiz-marquee/quiz-marquee.js

```diff
@@ -68,9 +68,17 @@ export default async function init(el) {
   [...rows].forEach(async (row) => {
     const cols = row.querySelectorAll(':scope > div');
     const isFragRow = cols[0].textContent.trim() === 'nested-fragments';
+    let isList = false;
+
+    if (cols[2]) {
+      if (cols[2].textContent.trim() === 'list') isList = true;
```

Comment:
Minor - Can we change to `if (cols[2].textContent?.trim() === 'list') isList = true;` to safeguard it?

Reply:
@sabyamon so there won't always be a cols[2] (default is not) - only if there is a content that needs to be a list. If I don't first wrap it in if (cols[2]), then then I get console errors that I'm trying to textContent.trim an invalid value.

Reply:
This would clean that up a bit to remove the nested if and safeguard the textContent.

Get rid of the `let` and use `const` like this:
```suggestion
     const isList = cols[2]?.textContent?.trim() === 'list' || false;
```

Give it a try on your local. It should return the same true/false values as needed.

cc: @sabyamon